intellij-plugin/README.md

```diff
@@ -20,13 +20,16 @@ An IntelliJ plugin supporting Arcs manifests (*.arcs files).
 
 1) Install IntelliJ
 
-2) From IntelliJ, import this directory as a Gradle Project.
+2) Install plugin: Grammar-Kit
```

Comment:
Do the versions of this plugin (Grammar-Kit) or JFlex matter? Like, should we state what versions this is/was developed with, in case later they change and are incompatible? OK if not but figured to ask while reading. I realize we're fundamentally building atop whatever JetBrains provides as their plugin development infra.

@piotrswigon I think the instructions in `## For Users` above on using `gradlew` directly will need updating as well, re: JFlex generation, not sure how to get Grammar-Kit for that workflow.

Reply:
I added more details instructions on step 2 - 5.